daml-lf/spec/daml-lf-1.rst

```diff
@@ -685,9 +685,15 @@ available for usage::
       ::= 'no_key'
        |  'key' τ eₖ eₘ
 
+  Temeplate choice observers
```

Comment:
```suggestion
  Template choice observers
```

Reply:
Thanks :) My keyboard likes to add extra 'e' :-/